### ISSUE TITLE
Enhancement idea

### ISSUE THREAD

**Issue by nnotestine** [mrk-its/homeassistant-blitzortung#115]
Please add use device location instead of only home location. It would be great if detection followed a users phone gps location.

**Comment by ypid** [comment]
@nnotestine Is this feature request covered by #27?

**Comment by nnotestine** [comment]
Yes! Appears to be exactly what I was asking for. I'm using distance to
lightning under 10 miles to alert. It would be great if it was based on my
phone location and not just home location.

**Comment by ypid** [comment]
@nnotestine Than you might want to close this/your issue as duplicate?

**Comment by nnotestine** [comment]
Marked as completed, duplicate.
